src/main/java/org/alfresco/repo/domain/schema/SchemaBootstrap.java

```diff
@@ -1645,6 +1729,10 @@ public synchronized void onBootstrap(ApplicationEvent event)
                     {
                         createdSchema = updateSchema(connection);
                         updatedSchema = true;
+                        if (logger.isInfoEnabled())
+                        {
+                            logger.info("Created schema value: " + createdSchema);

```

Comment:
This either needs to be a debug or say something that it is a fresh installation vs upgrade process.